### PR TITLE
Customize a node docs enhancement

### PR DESCRIPTION
**Pull request by krmahadevan**
* Added details on how node types
* Made some formatting changes to the docs.

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.

Enhanced the docs for customising a node and added details on how to use a regular jar via `--ext` argument.
Also added details on Node implementation types and their relevance

### Types of changes

- [X] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [X] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [X] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/grid/advanced_features/customize_node.en.md [hunk 1]
```diff
@@ -3,13 +3,11 @@ title: "Customizing a Node"
 linkTitle: "Customize Node"
 weight: 4
 aliases: [
-"/documentation/en/grid/grid_4/customize_node/",
-"/documentation/grid/advanced_features/customize_node/"
 ]
```
Comment (diemol) [inline website_and_docs/content/documentation/grid/advanced_features/customize_node.en.md]: The whole `aliases` section should be removed.

**Review by diemol — CHANGES_REQUESTED** [review]
Left some comments, thank you.

File: website_and_docs/content/documentation/grid/advanced_features/customize_node.en.md (hunk 1 above)
Comment (krmahadevan) [reply website_and_docs/content/documentation/grid/advanced_features/customize_node.en.md]: Fixed and pushed the changes. Please check @diemol

**Review by diemol — APPROVED** [review]
Thank you, @krmahadevan!

**Pull request merged**